### ISSUE TITLE
Resizing Subtitles

### ISSUE THREAD

**Issue by pepijn809** [senpaiski/TopAnimeStream#23]
A Future Request. 

In the app subtitles become larger when in full screen, this does not happen as of yet in the website. This is something that can be changed in the future. 

My question however remains, are there plans (future plans) to include options to resize subtitles to your need, cross platform?

**Comment by senpaiski** [comment]
Yes! I could create a modal with subtitle settings and you will be able to change the subtitle the way you want.
- Presets
- Change size of subtitles
- Change text color
- Change background color
- Change font type (with import)

**Comment by Draxler** [comment]
:+1:

**Comment by pepijn809** [comment]
Cool, that would be rather useful :)
